repos/system_upgrade/el7toel8/models/dnfplugintask.py

```diff
@@ -0,0 +1,16 @@
+from leapp.models import Model, fields
+from leapp.topics import SystemInfoTopic
+
+
+class DNFPluginTask(Model):
+    """
+    Represents information about what DNF plugin and in which stage
+    to disable.
+
+    Available stages: "check", "download" and "upgrade
```

Comment:
What about something like (just nitpick, not blocker):
```suggestion
    Represents information what should DNF do with a specifiec DNF plugin in various stages.
    
    Currently, it's possible just to disable specified DNF plugins.
    Available stages: "check", "download" and "upgrade
```